src/main/java/lotto/domain/LottoGroup.java

```diff
@@ -0,0 +1,16 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class LottoGroup {
+
+    private final List<Lotto> lottoGroup;
+
+    public LottoGroup(List<Lotto> lottoGroup) {
+        this.lottoGroup = lottoGroup;
+    }
+
+    public List<Lotto> getLottoGroup() {
+        return lottoGroup;
```

Comment:
```suggestion
        return Collections.unmodifiableList(lottoGroup);
```
unmodifiableList를 사용하면 어떤 장점이 있을까요?